deno/compress_image/src/mod.ts

```diff
@@ -0,0 +1,77 @@
+import { base64Encode } from "./deps.ts";
+
+export default async function (req: any, res: any) {
+  const { provider, image } = JSON.parse(req.payload);
+  switch (provider) {
+    case "tinypng":
+      if(!req.variables.TINIFY_API_KEY) {
+        return res.json({
+          success: false,
+          message: "The environment variables (TINIFY_API_KEY) are required."
+        })
+      }
+
+      try {
+        const receiveImage = await fetch(image);
+        const requestImage = await fetch("https://api.tinypng.com/shrink", {
+          method: "POST",
+          headers: {
+            "Accept": "*/*",
+            "Cache-Control": "no-cache",
+            "Authorization": `Basic ${base64Encode("api:" + req.variables.TINIFY_API_KEY)}`
+          },
+          body: await receiveImage.body
+        });
+        
+        return res.json({ 
+          success: true, 
+          image: (await requestImage.json()).output.url
+        });
+      } catch (error) {
+        return res.json({
+          success: false,
+          message: error
```

Comment:
If an error occurs, I get a response like this:

```json
{"response":{"success":false,"message":{}},"stdout":"","stderr":""}
```

It seems like `error` isn't serialized correctly. Let's try:

```suggestion
          message: error.toString()
```